<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feature Inspector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutLeft">
      <item>
       <widget class="InspectionPlotWidget" name="graphWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutRight">
      <property name="topMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableFeatureBounds">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>250</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>86</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>86</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <column>
         <property name="text">
          <string>Blue Low</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Blue High</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Red Low</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Red High</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutMeasurements">
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelPreviousFeature">
          <property name="text">
           <string>Last Feature:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelPew">
          <property name="text">
           <string>PEW:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelCurrentVelocityErr">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="labelCurrentPewError">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelCurrentVelocity">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelVelocity">
          <property name="text">
           <string>Velocity:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditFeatureEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelPreviousFeatureStart">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelPreviousFeatureEnd">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditFeatureStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelCurrentPew">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelFeatureRange">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Feature Range:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutNotes">
        <item>
         <widget class="QLabel" name="labelNotes">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkboxFlagSpectrum">
          <property name="text">
           <string>Flag spectrum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkboxFlagFeature">
          <property name="text">
           <string>Flag Feature</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditNotes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="pushButtonCalculate">
          <property name="toolTip">
           <string extracomment="Measure feature using current bounds"/>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="toolTip">
           <string extracomment="Save current feature measurements"/>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPrevious">
          <property name="toolTip">
           <string extracomment="Previous feature"/>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonNext">
          <property name="toolTip">
           <string extracomment="Next feature"/>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFinished">
          <property name="toolTip">
           <string extracomment="Plot the next Spectrum"/>
          </property>
          <property name="text">
           <string>Finished</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1387</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionResetPlot"/>
    <addaction name="actionViewResults"/>
   </widget>
   <widget class="QMenu" name="menuSkip">
    <property name="title">
     <string>Skip</string>
    </property>
    <addaction name="actionNextFeature"/>
    <addaction name="actionLastFeature"/>
    <addaction name="separator"/>
    <addaction name="actionNextSpectrum"/>
    <addaction name="actionPreviousSpectrum"/>
    <addaction name="separator"/>
    <addaction name="actionNextSN"/>
    <addaction name="actionPreviousSN"/>
    <addaction name="separator"/>
    <addaction name="actionGoTo"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFeatureDefinitions"/>
    <addaction name="actionAnalysisSettings"/>
    <addaction name="actionPlottingStyle"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSkip"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionResetPlot">
   <property name="text">
    <string>Reset Plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFeatureDefinitions">
   <property name="text">
    <string>Feature Definitions...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPlottingStyle">
   <property name="text">
    <string>Plotting Style...</string>
   </property>
  </action>
  <action name="actionAnalysisSettings">
   <property name="text">
    <string>Analysis Settings...</string>
   </property>
  </action>
  <action name="actionViewResults">
   <property name="text">
    <string>View Results...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNextSpectrum">
   <property name="text">
    <string>Next Spectrum</string>
   </property>
  </action>
  <action name="actionPreviousSpectrum">
   <property name="text">
    <string>Previous Spectrum</string>
   </property>
  </action>
  <action name="actionNextSN">
   <property name="text">
    <string>Next SN</string>
   </property>
  </action>
  <action name="actionPreviousSN">
   <property name="text">
    <string>Previous SN</string>
   </property>
  </action>
  <action name="actionGoTo">
   <property name="text">
    <string>Go To...</string>
   </property>
  </action>
  <action name="actionNextFeature">
   <property name="text">
    <string>Next Feature</string>
   </property>
  </action>
  <action name="actionLastFeature">
   <property name="text">
    <string>Last Feature</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InspectionPlotWidget</class>
   <extends>QWidget</extends>
   <header>leed.app.widgets.inspectionPlotWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
